javascript/new-src/src/js/app.js

```diff
@@ -713,18 +733,24 @@ angular
       $scope.active = [];
       $scope.enabled = [];
       $scope.getInfo = function() {
-        $ola.get.ItemList().then(function(data) {
-          $scope.Items = data;
-        });
+        $ola.get.ItemList()
+          .then(function(data) {
+            $scope.Items = data;
+          });
       };
       $scope.getInfo();
       $scope.Reload = function() {
-        $ola.action.Reload().then();
+        $ola.action.Reload();
         $scope.getInfo();
       };
       $scope.go = function(id) {
         $location.path('/plugin/' + id);
       };
+      $scope.changeStatus = function(id, current) {
+        $ola.post.PluginState(id, current);
+        $scope.Reload();
```

Comment:
We should do a better job on the server side of handling this. We shouldn't be reloading all the plugins just to change the state of a single plugin.  

@peternewman  or I can make the change on the server side so you can remove this call.